util-core/src/test/scala/com/twitter/util/VarTest.scala

```diff
@@ -36,7 +36,7 @@ class VarTest extends FunSuite with GeneratorDrivenPropertyChecks {
     assert(Var.sample(s) == "8923")
 
     var buf = mutable.Buffer[String]()
-    s.changes.register(Witness({ v => buf += v }))
+    s.changes.register(Witness({ (v: String) => buf += v; {} }))
```

Comment:
do you mind using () instead of {}?  doesn't really matter, but it's the style we normally use in util.